buildlogic/src/main/kotlin/com/tidal/sdk/plugins/KotlinAndroidApplicationConventionPlugin.kt

```diff
@@ -72,15 +74,19 @@ internal class KotlinAndroidApplicationConventionPlugin : Plugin<Project> {
                 compose = true
             }
         }
-        val ensureClientIdPresent: TaskProvider<*> = tasks.register("ensureClientIdPresent") {
```

Comment:
doesn't this here mean that now you explicitly DO check for `clientSecret`?
So. now it wouldn't be required for running, but is required for building?

Reply:
Certainly poor wording on my end I'm noticing. But it's basically what you're saying yes - in base it's intended to be possible to use the app without a secret, but it works slightly differently so we need to check for the presence of one in the diff in the other file to decide behavior. Now this needs to become no longer possible, so instead of checking for presence of the secret at runtime, we force it to be present at build time.